src/components/sidebar-image-grid/SidebarImageGrid.styles.ts

```diff
@@ -0,0 +1,20 @@
+export const styles = {
+  imageGrid: {
+    display: 'flex',
+    gap: '13px',
+    px: '16px',
+    justifyContent: 'start'
+  },
+  imageWrapper: {
+    m: '20px 10px'
+  },
+  modalImage: {
+    width: '100%',
+    borderRadius: '5px'
+  },
+  modalImageGrid: {
+    display: 'grid',
+    gridTemplateColumns: { xs: '1fr 1fr 1fr 1fr', md: '1fr 1fr 1fr' },
```

Comment:
gridTemplateColumns: { xs: 'repeat(4, 1fr)', md: 'repeat(3, 1fr)' },

Reply:
> gridTemplateColumns: { xs: 'repeat(4, 1fr)', md: 'repeat(3, 1fr)' },

that's much better, fixed